RecoPPS/Local/plugins/CTPPSPixelLocalTrackProducer.cc

```diff
@@ -65,31 +65,31 @@ class CTPPSPixelLocalTrackProducer : public edm::stream::EDProducer<> {
 
   edm::InputTag inputTag_;
   edm::EDGetTokenT<edm::DetSetVector<CTPPSPixelRecHit>> tokenCTPPSPixelRecHit_;
```

Comment:
why not to make `const` all other classs members initialized in the ctor initialization list (and never modified afterwards)?